src/main/java/org/folio/consortia/service/impl/UserAffiliationServiceImpl.java

```diff
@@ -38,6 +39,7 @@ public class UserAffiliationServiceImpl implements UserAffiliationService {
 
   @Override
   @SneakyThrows
+  @Transactional
```

Comment:
Lets will not wrap entire method within the catch block due these 2 reasons:

- if we make some changes in DB but after this sending CONSORTIUM_PRIMARY_AFFILIATION_CREATED event to kafka will  be failed - we will remain system in the inconsistent state. When we remove catch block -> exception will be thrown and spring will roll back the transaction automatically. I propose use try/catch only for parsing json payload
- if exception thrown - message offset willl be not commited and it will give us chance for retry

Reply:
removed try and catch.